apps/src/templates/projects/deleteDialog/DeleteProjectDialog.story.jsx

```diff
@@ -17,6 +18,7 @@ export default storybook => {
             isDeletePending={false}
             projectId={PROJECT_ID}
             onClose={action('close')}
+            deleteProject={() => console.log("Delete project")}
```

Comment:
not high priority, but you can use `action('deleteProject')` here and on line 34 like you did on line 20